### ISSUE TITLE
[A11Y] Android: Responsive Menubar component have hidden graphic on it

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#388]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet):  Nexus 9
Browser: Chrome Dev 59.0.3062.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to COMPONENTS link and double tap on it.
3. Linearly navigate to the “Navigation” components section and linearly navigates through the Responsive Menubar.

**ACTUAL RESULTS:**

While linearly navigating a hidden image is encountered and talkback says it as “The blog, graphic”

**EXPECTED RESULTS:**

No hidden content should be encountered or identified when using screen reader.

**ATTACHMENT(S):**

<img width="1440" alt="hidden graphic" src="https://cloud.githubusercontent.com/assets/26986019/24921752/bae4d9a2-1ea0-11e7-85e0-8d33ed079215.png">

**Comment by torch2424** [comment]
Fixed in my a11y bugs branch

[Commit Fix](https://github.com/torch2424/ampstart/commit/53d3fff71d52af0d00dbb1bf6348030ce3b35101)
